### PR TITLE
Docs 6834 b2 b connector update AS2 mc

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

**Review by bharathcp — APPROVED** [review]

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc [hunk 1]
```diff
@@ -31,10 +35,10 @@ You should have familiarity with AS2.
 
 The AS2 version 5 connector requires the `Async MDN` to be handled as a separate flow. The retry of AS2 Message Send is not automatic. The application has to handle the retry upon failure in an MDN or upon failure to receive an MDN.
 
-Version 5 enables you to reuse the application HTTP listener configuration.
-
+Version 5 enables you to reuse the application HTTP listener configuration. Refer to xref:as2/AS2-connector-upgrade-migrate.adoc[Upgrade and Migrate] for more information.
```
Comment (valkyrie69) [inline modules/ROOT/pages/as2/as2-connectorMule4.adoc]:
```suggestion
Version 5 enables you to reuse the application's HTTP listener configuration. Refer to xref:as2/AS2-connector-upgrade-migrate.adoc[Upgrade and Migrate] for more information.
```

File: modules/ROOT/pages/as2/as2-connector-reference.adoc [hunk 2]
```diff
@@ -373,7 +375,6 @@ non-repeatable-stream |  Configure to use repeatable streams. |  |
 * ListenerConfig
 
 
-
 [[as2-mdn-listener]]
 === As2 MDN Listener
```
Comment (valkyrie69) [inline modules/ROOT/pages/as2/as2-connector-reference.adoc:379]: AS2  Do a S&R for these. I know it's generated...

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc (hunk 1 above)
Comment (valkyrie69) [inline modules/ROOT/pages/as2/as2-connectorMule4.adoc]:
```suggestion
AS2 Connector version 5.0.0 enables you to reuse the application's HTTP listener configuration. Refer to xref:as2/AS2-connector-upgrade-migrate.adoc[Upgrade and Migrate] for more information.
```

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc [hunk 3]
```diff
@@ -31,10 +35,10 @@ You should have familiarity with AS2.
 
 The AS2 version 5 connector requires the `Async MDN` to be handled as a separate flow. The retry of AS2 Message Send is not automatic. The application has to handle the retry upon failure in an MDN or upon failure to receive an MDN.
```
Comment (valkyrie69) [inline modules/ROOT/pages/as2/as2-connectorMule4.adoc:36]: This paragraph that starts on line 36 now is confusing. I'm not sure what it is saying.  Maybe:
AS2 Connector 5.0.0 requires `Async MDN` to be handled as a separate flow, as AS2 Message Send retry is not automatic. The application must handle retry upon failure in an MDN, or upon failure to receive an MDN.

Comment (valkyrie69) [reply modules/ROOT/pages/as2/as2-connectorMule4.adoc:36]: I really don't know what it's saying though. An MDN is like a return receipt. So how is retry upon failure handled in an MDN?

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/as2/as2-connectorMule4.adoc]: Fixed.

File: modules/ROOT/pages/as2/as2-connector-reference.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/as2/as2-connector-reference.adoc:379]: Fixed where I could.

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/as2/as2-connectorMule4.adoc]: Fixed.

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/as2/as2-connectorMule4.adoc:36]: These should be bullets. Fixed.

**Pull request merged**